crates/beerus-rpc/src/lib.rs

```diff
@@ -69,6 +70,24 @@ impl BeerusApiServer for BeerusRpc {
             .map_err(|_| Error::from(BeerusApiError::InternalServerError))
     }
 
+    async fn ethereum_get_block_transaction_count_by_number(
+        &self,
+        block_tag: &str,
```

Comment:
Let's accept the argument as a BlockTag instead of creating it in the function body